.ci/scripts/release/prepare.sh

```diff
@@ -2,11 +2,16 @@
 
 # update apt repositories and install missing utilities
 # add GitHub packages Debian repository for gh
-curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | \
-      tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+
 apt update
-apt install -y gpg gh
+apt install -y gpg git
```

Comment:
:x: It's not required to install git, we were previously only installing `gpg`.

Reply:
Where is the existing git coming from? I added it because `gh` has a dependency to `git` and assumed previously it was installed automatically by apt when installing `gh`.

Reply:
I would assume it's already in the image? You can look at the revision before we added `gh`, and we were only installing `gpg` (but using `git` already)

Reply:
You are right. The base image is `maven:3.8.4-eclipse-temurin-17` and it already contains git. So I will remove installing git and the setting of safe directory. Hope this will work next time :)